skrub/_table_vectorizer.py

```diff
@@ -368,13 +313,10 @@ class TableVectorizer(TransformerMixin, BaseEstimator):
         ``len(transformers_)==len(transformers)+1``, otherwise
         ``len(transformers_)==len(transformers)``.
 
-    types_ : dict mapping of int to type
+    type_per_column_ : dict mapping of int to type
         A mapping of inferred types per column.
         Key is the index of a column, value is the inferred dtype.
-        Exists only if `auto_cast=True`.
-
-    imputed_columns_ : list of str
-        The list of columns in which we imputed the missing values.
+        Exists only if ``activate_auto_cast=True``.
```

Comment:
it should exist in all cases, when autocast is disabled then it will just be the input's dtypes